hack/tools/go.mod

```diff
@@ -7,7 +7,7 @@ replace sigs.k8s.io/cluster-api => ../../
 replace sigs.k8s.io/cluster-api/test => ../../test
 
 require (
-	cloud.google.com/go/storage v1.29.0
+	cloud.google.com/go/storage v1.30.1
```

Comment:
Note: these and below deps were not bumped intentionally but automatically while running `go mod tidy` in the hack/tools folder